docs/quickstart/quickstart_staged_tree_notebook.ipynb

```diff
@@ -0,0 +1,193 @@
+    "- `Merged Situations` - a list of tuples representing the partition of the nodes into stages\n",
+    "- `Log Likelihood` - the log likelihood of the data under the model selected by AHC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cegpy import StagedTree\n",
+    "\n",
+    "st = StagedTree(df)\n",
+    "st.calculate_AHC_transitions()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Within `cegpy`, singleton stages, i.e. stages contatining a single situation, are coloured white, leaves and their corresponding sink node are coloured in light-grey. Runing AHC on our data set results in the following staged tree."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "st.create_figure()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Structural Zeros"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Custom Hyperstages"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Modifiying the Graph of a StagedTree\n",
+    "\n",
+    "By default, the colours of the nodes in `cegpy` are selected uniformly at random from the entire spectrum of colours. If we want to use a specific colour palette, a list of colours to be used by the AHC algorithm can be specified as a parameter when calling the `create_AHC_tarnsitions` method, for example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "colours = ['#BBCC33','#77AADD','#EE8866','#EEDD88','#FFAABB','#44BB99']\n",
+    "st.calculate_AHC_transitions(colour_list=colours)\n",
+    "st.create_figure()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`cepgy` is built on [NetworkX](https://networkx.org) adn [PyDotPlus](https://pydotplus.readthedocs.io), we can access the underlying `pydotplus.grphiviz.Dot` object by accesing the  `dot_graph` property. This enables visual modifications of our staged tree or CEG. For example, the following code modifies the distance between the nodes and changes the style of an edge from solid to dashed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image\n",
+    "g = st.dot_graph()\n",
+    "g.set('ranksep', 1)\n",
+    "g.set('nodesep', 0.005)\n",
+    "g.get_edge(('s8', 's20'))[-1].set_style('dashed')\n",
```

Comment:
This is really cool! 

One issue I do have is that it is not clear what this is doing unless you dig into the api.

Something like this (although much more verbose) would be clearer to the reader.
```python
for edge in g.get_edge(src_or_list='s8', dst='s20'):
    if "Hard" in edge.get("label"):
        edge.set_style('dashed')
```

This makes it clear that `get_edge` it is returning a list, and that you intend to modify that specific edge.

Reply:
Oh yes I agree!